foyer/tests/implemented_opls_tests.txt

```diff
@@ -72,3 +72,5 @@ toluene
 trichloromethane
 2-methoxypropane
 methyltert-butylether
+tbutanol
```

Comment:
I think this is supposed to be `t-butanol`, but I'm not sure

Reply:
it is t-butanol.